docs/getting-started/advanced-setting.asciidoc

```diff
@@ -71,9 +71,9 @@ collectors to ship data to {es}, the data must be mapped to ECS.
 [[update-threat-intel-indices]]
 == Update default Elastic Security threat intelligence indices
 
-The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat indicators. The setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
+The `securitySolution:defaultThreatIndex` advanced setting specifies threat intelligence indices that {elastic-sec} features query for ingested threat indicators. This setting affects features that query threat intelligence indices, such as the Threat Intelligence view on the Overview page, indicator match rules, and the alert enrichment query. You can specify one or more threat intelligence indices; multiple indices must be separated by commas. By default, only the `logs-ti*` index pattern is specified. {agent} integrations use this index pattern.
```

Comment:
Does the last sentence mean that the `logs-ti*` pattern is required? If so, maybe be a little more explicit that the user shouldn't remove or overwrite it if they want to use Elastic Agent integrations.

Reply:
Yes, it's required for using Elastic Agent integrations. How about:
By default, only the `logs-ti*` index pattern is specified. Do not remove or overwrite this index pattern, as it's used by {agent} integrations.

Reply:
LGTM!